inference-engine/tests/functional/plugin/gna/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -9,27 +9,8 @@
 
 std::vector<std::string> disabledTestPatterns() {
     return {
-        ".*TensorNamesTest\\.CheckAddOutput.*",
-        // TODO: FIX BUG 31661
-        // TODO: support InferRequest in GNAPlugin
-        ".*InferRequestTests\\.canRun3AsyncRequestsConsistentlyFromThreadsWithoutWait.*",
```

Comment:
Well, this is failing right now on master branch. Could you please return it back?

Reply:
How is it possible that pre-commit is passed for this PR?